### PR TITLE
(QENG-1721) added installation support of dev versions of windows puppet...

### PR DESCRIPTION
**Pull request by kevpl**
...-agent

### CODE REVIEW

File: lib/beaker/dsl/install_utils.rb [hunk 1]
```diff
@@ -1279,6 +1284,10 @@ def install_puppetagent_dev_repo( host, opts )
         when /^(debian|ubuntu|cumulus)$/
           on host, "dpkg -i --force-all #{onhost_copied_file}"
           on host, "apt-get update"
+        when /^windows$/
+          result = on host, "echo #{onhost_copied_file}"
```
Comment (anodelman) [inline lib/beaker/dsl/install_utils.rb]: There is a built in host.echo command - you should use that.

File: lib/beaker/dsl/install_utils.rb [hunk 2]
```diff
@@ -1279,6 +1284,10 @@ def install_puppetagent_dev_repo( host, opts )
...
+          onhost_copied_file = result.raw_output.chomp
+          on host, "cmd /C 'start /w #{onhost_copied_file}'"
```
Comment (anodelman) [inline lib/beaker/dsl/install_utils.rb]: If you use {:cmdexe => true} in a command object it will correctly append 'cmd /C' to the command string.

Comment (joshcooper) [inline lib/beaker/dsl/install_utils.rb]: I'm also not sure about the need for single quotes, we are not currently doing that in puppet: https://github.com/puppetlabs/puppet/blob/master/acceptance/lib/puppet/acceptance/install_utils.rb#L252

**Comment by kevpl** [comment]
review feedback included

**Comment by kevpl** [comment]
just the debian foss testing errors

**Comment by anodelman** [comment]
Could you back fill some spec tests for this?

**Pull request merged**